packages/consts/src/consts.ts

```diff
@@ -111,7 +116,25 @@ export const USERNAME = {
 
     // Regex matching a potentially allowed username. The numbers must match MIN and MAX!
     // Note that username must also pass isForbiddenUser() test to be allowed!
-    REGEX: /^[a-zA-Z0-9_.-]{3,30}$/,
+    REGEX: /^[a-zA-Z0-9_-]{3,30}$/,
```

Comment:
For a clarification, excluding the dot makes this regex effectively exclude any viable URLs and email addresses.

Reply:
We have quite a few users who have domains as usernames (the definitely valid ones are usually the domain of the company).

And it seems that profile.name is a similar case.

Reply:
As it will be implemented in the apify-core (see draft [here](https://github.com/apify/apify-core/pull/18188)), existing users (nor their usernames nor profile names) will not be affected till they decide to modify their data.
It should affect them only when attempt to create new user (segmentation) / organization, update username or profile and to convert user to organization is made.

Reply:
Thanks for the comment. I checked on the web and we use the function in two kinds of pages: actor detail (e.g. https://apify.com/trudax/reddit-scraper) and user detail (e.g. https://apify.com/trudax) => when there is the username in the url.
If the username is deemed "not valid", we send back a 404 Not Found.

So the problem should be clear: if we have any users (and their actors) that use the dot, we will give an error on their page even if their username was valid at the time when they chose it. From a quick look to the store I see that a lot of users have dots in their usernames

So my question is: is any action going to be taken for the users that already have the bad usernames? If not, then I think it's acceptable that in console we keep using the old regex, since we only have the two usecases mentioned above

Reply:
Personally, I'd keep the dot, and just deal with "proper" links.

Reply:
Yes, that would be a viable solution for a web (as the regex there is not used for any mutations).

If we are willing to live with some links (urls) getting through (and therefore being rendered as links by Slack, Gmail and such), I have no problem with keeping the dot in both username and profile names (org, first, last, ...) as suggested by @valekjo.

@gippy, what do you think ?

Reply:
I believe the dot check is too strict and it should be enough to prevent users from intentionally writing URLs. If a email client converts the name to url because it contains `.` it's outside of our control.

Reply:
@baldasseva, with this decision, no need for any change in apify-web.